src/System.Globalization/tests/Performance/Perf.StringCultureInfo.cs

```diff
@@ -0,0 +1,106 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
```

Comment:
This file won't be merged, but I did have a question on it. My builds for it were failing with the notice that the overloads I'm trying to use (ones that take a CultureInfo) didn't exist. Any ideas?